src/Readme.txt

```diff
@@ -86,6 +86,7 @@ https://www.billmate.se/press-och-media/
 
 =  3.9.2 (2020-01-23)
 * Fix - Resolved debug errors being generated by module 
+* FIx -Resolved critical error in library, caused by wordpress not being able to connect to billmates api
```

Comment:
Fix - Resolved critical error in library, caused by Wordpress not being able to connect to Billmates API